common/cards/hermits/index.ts

```diff
@@ -157,21 +157,21 @@ export const HERMIT_CARD_CLASSES: Array<HermitCard> = [
 	new RenbobRareHermitCard(),
 
 	//Advent calendar cards
```

Comment:
It was possibly my fault, but do you mind ensuring all indexes are commented out